src/components/DirectLinks/DirectLinks.module.scss

```diff
@@ -0,0 +1,62 @@
+@import 'styles/_global.scss';
+
+.DirectLinks {
+  width: $main-max-width;
+
+  h2 {
+    @include heading-large;
+    margin-bottom: 1.6rem;
+  }
+}
+
+.InnerContainer {
+  width: $main-boxed-width;
+  margin: 0 auto;
+}
+
+.LinkList {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+
+  li {
+    border-bottom: 0.1rem $color-neutral-grey2 solid;
+    line-height: 5.5rem;
+    float: left;
+  }
+
+  li:nth-child(odd) {
+    width: calc(calc(#{$main-boxed-width} / 2) - 2rem);
```

Comment:
Can't you just use `width: calc(50% - 2rem)` in the `li { ... }` above?

Reply:
Also, instead of floating, you can also do: `columns: 2` in the ul. Possibly make the `li` `inline-block`

Reply:
Hmm, having some issues with styling then. I'm leaving it as it is. I did simplify the width.